src/main/kotlin/org/jitsi/nlj/RtpReceiver.kt

```diff
@@ -40,6 +41,8 @@ abstract class RtpReceiver :
 
     abstract fun setAudioLevelListener(audioLevelListener: AudioLevelListener)
 
+    abstract fun onBandwidthEstimateChanged(listener: BandwidthEstimator.Listener)
```

Comment:
I don't love this name for this function - it sounds to me like what gets called when the bandwidth estimate changes, not what you call to add a listener.  Maybe `addBandwidthEstimateListener`?

Reply:
Had I not just merged it, I would have made the change :)